nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-client-service-api/src/main/java/org/apache/nifi/mongodb/MongoDBClientService.java

```diff
@@ -31,10 +31,13 @@
 public interface MongoDBClientService extends ControllerService {
      String WRITE_CONCERN_ACKNOWLEDGED = "ACKNOWLEDGED";
      String WRITE_CONCERN_UNACKNOWLEDGED = "UNACKNOWLEDGED";
-     String WRITE_CONCERN_FSYNCED = "FSYNCED";
+     String WRITE_CONCERN_FSYNCED = "FSYNCED (Deprecated)";
```

Comment:
I think changing the string  of the write concerns will create an upgrade incompatibility. If we want to communicate the deprecation of these we can create an `AllowableValue` object for these with a description of the deprecation and a note about the replacement.